core/storage/base_model/gae_models.py

```diff
@@ -538,10 +540,14 @@ def get_multi_versions(cls, entity_id, version_numbers):
             version_numbers: list(int). List of version numbers.
 
         Returns:
-            list(VersionedModel|None). Model instances representing the given
+            list(VersionedModel). Model instances representing the given
                 versions.
         """
         instances = []
+        try:
```

Comment:
This is not robust. What if the entity with the given id exists but at least one of the version numbers is higher than the current version?

Reply:
Done.